pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -194,6 +195,7 @@ decl_event!(
         NodeUpdated(types::Node),
         NodeDeleted(u32),
         NodeUptimeReported(u32, u64, u64),
+        NodeUsedResourcesReported(types::Resources),
```

Comment:
Add this event at the end of the enum, so the index does not change for the other events in the enum

see other comment as well